services/common/src/redux/rootState.ts

```diff
@@ -14,6 +18,22 @@ export const getStore = (preloadedState = {}) =>
 
 export const store = getStore();
 
+/**
+ * Provide typed versions for global redux functions
+**/
+
 type RootState = ReturnType<typeof store.getState>;
 
+type AppAction = UnknownAction | FormAction;
+
+// Infer the `RootState` and `AppDispatch` types from the store itself + make sure redux-form acti are captured as well.
+export type AppDispatch = typeof store.dispatch & Dispatch<AppAction>;
```

Comment:
Sonarcloud doesn't like this `&` type. Unfortunately needed as `AppAction` is coming from `redux-form` and couldn't find a way of avoiding it without providing our own `redux-form` typing